src/main/java/com/softserveinc/speakukrainian/pageobjects/BasePage.java

```diff
@@ -18,10 +18,10 @@ public void pressArrowDown(int count) {
         for (int i=0; i<count; i++) {
             actions()
                     .sendKeys(Keys.ARROW_DOWN)
-                    .sendKeys(Keys.ENTER)
                     .build()
                     .perform();
         }
+        actions().sendKeys(Keys.ENTER).build().perform();
```

Comment:
The method used to press the 'down arrow' and 'enter' a specified number of times, now the method presses the 'arrow' a specified number of times and presses 'enter only once after pressing the arrows